src/parser/AttributeValueParser.java

```diff
@@ -0,0 +1,38 @@
+package parser;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class AttributeValueParser {
+    private Object value;
+    private static final Pattern ID_PATTERN = Pattern.compile(" Id :(.)+");
+    private static final Pattern COLOR_PATTERN = Pattern.compile("Color :(.)+");
+    private static final String ID = "Id";
+    private static final String COLOR = "Color";
+    private static final Integer SIX = 6;
```

Comment:
What is 6 and 8? These variables are need to be named understandable. e.g. ID_GROUP_INDEX or COLOR_GROUP_INDEX